markdown/events/2022/hackathon-october-2022.md

```diff
@@ -142,9 +241,112 @@ Keep an eye out for others hosting community events in your area. We will be sha
+- Finally, on Thursday, we will be running a short quiz!
+
+All social activities are of course optional, but hope to see as many people joining in as possible :tada:
+
 # COVID control
 
 Please remember that the COVID-19 pandemic is not over.
 If you intend on joining the hackathon in person, we request that you get vaccinated and tested, if possible. We ask for anyone who displays symptoms to not mingle with other participants. Whilst it is not feasible for us to enforce any rules, we request that attendees behave responsibly.
 
 We will be monitoring the status of the pandemic as the event draws closer and reserve the right to make changes in our policies if we deem that to be necessary.
+
+# Safety Information at the Hackathon
+
+Please note that our [Code of Conduct](https://nf-co.re/code_of_conduct) applies to the Hackathon, and all participants need to abide by our guidelines to participate. We should all feel responsible for making nf-core events safe and fun for everyone.
+
+In addition, please respect the following at all times during the hackathon:
+
+- **Never share the jitsi and zoom links** for the hackathon on social media.
+- Do not take screenshots of groups you are working within without asking for consent of all individuals in the group. Remember, not everyone wants their photographs shared on social media. Not respecting the latter amounts to doxing, which is also in direct violation of the nf-core [Code of Conduct](https://nf-co.re/code_of_conduct).
+
+## Reporting CoC Violations during the hackathon
+
+In case of an immediate perceived threat at the hackathon, please reach out to any of the following individuals on Slack. Their names and the time-zones they are located in are indicated.
+
+- Alexander Peltzer (CET)
+- Harshil Patel (GMT)
+- Phil Ewels (CET)
+- Gisela Gabernet (CET)
+- James A. Fellows Yates (CET)
+- Matthias Hörtenhuber (CET)
+- Maxime Garcia (CET)
+- Renuka Kudva (CET)
+- Kevin Menden (CET)
+- Friederike Hanssen (CET)
+- Michael Heuer (PST)
+
+You can also report any CoC violations directly to safety [at] nf-co [dot] re. Our safety officer, Renuka Kudva, will contact you to follow up on your report.
+
+In your email report, please do your best to include:
+
+- Your contact information.
+- Identifying information (e.g. names, nicknames, pseudonyms) of the participant who has violated the Code of Conduct.
+- The behaviour that was in violation and the circumstances surrounding the incident.
+- The approximate time of the behaviour (if different than the time the report was made)
+- Other people involved in the incident if applicable.
+- If you believe the incident is ongoing, please let us know.
+- If there is a publicly available record (e.g. mailing list record, a screenshot - please attach if so).
+- Any additional helpful information.
+
+After you file a report, uur safety officer, Renuka Kudva, will contact you to follow up on your report.
+
+### Who will read and handle reports?
```

Comment:
This stuff is all in the CoC already right? I think we can delete from here onwards..